package.json

```diff
@@ -5,12 +5,12 @@
   "author": "Jeff Dickey @jdxcode",
   "bugs": "https://github.com/oclif/config/issues",
   "dependencies": {
+    "@oclif/errors": "^1.2.2",
```

Comment:
"@oclif/errors": "^1" might hopefully reduce multiple versions

Reply:
I think it's better to encourage updating libraries using older versions of such packages, but I've changed the constraint to `^1.0.0` for the sake of getting this across the line.